src/utils/init/utils.js

```diff
@@ -16,8 +17,26 @@ const normalizeDir = ({ siteRoot, dir, defaultValue }) => {
   return relativeDir || defaultValue
 }
 
-const getDefaultSettings = ({ siteRoot, config }) => {
-  const { command: defaultBuildCmd, publish: defaultBuildDir, functions: defaultFunctionsDir } = config.build
+const getFrameworkDefaults = async ({ siteRoot }) => {
+  const frameworks = await listFrameworks({ projectDir: siteRoot })
+  if (frameworks.length !== 0) {
+    const [
+      {
+        build: { directory, commands },
+      },
+    ] = frameworks
+    return { frameworkBuildCommand: commands[0], frameworkBuildDir: directory }
+  }
+  return {}
+}
+
+const getDefaultSettings = async ({ siteRoot, config }) => {
+  const { frameworkBuildCommand, frameworkBuildDir } = await getFrameworkDefaults({ siteRoot })
+  const {
+    command: defaultBuildCmd = frameworkBuildCommand,
+    publish: defaultBuildDir = frameworkBuildDir,
+    functions: defaultFunctionsDir,
+  } = config.build
```

Comment:
Is it common for `config.build.command` or `config.build.publish` to be defined?
If so, would it make sense to compute the default value only when needed (for performance reason due to the time required to run `listFrameworks()`?

Reply:
As always another great observation.
The call is redundant when running `npm init` and linking to an existing that has both of those settings configured in the UI or there is a `netlify.toml` with both of those configured.

I'm not sure how often those happen with a new repo, but I think we can keep the call to `listFrameworks` for 2 reasons:
1. Code simplicity
2. We'll need to run `listFrameworks` always soon to recommend the Next.js plugin